app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)     
+    if @user.save
+      flash["success"] = "Your account has been created" \
+      redirect_to login_path
+    else
+      render :new
+    end
+  end
+
+  def user_params
+    params.require(:user).permit!
```

Comment:
This is very dangerous. If someone adds admin=1 to their post, they are now an admin. You definitely want to use strong parameters.